models/baseModels/SetupWizard/SetupWizard.ts

```diff
@@ -21,6 +21,21 @@ export function getCOAList() {
     { countryCode: 'ni', name: 'Nicaragua - Catalogo de Cuentas' },
     { countryCode: 'nl', name: 'Netherlands - Grootboekschema' },
     { countryCode: 'sg', name: 'Singapore - Chart of Accounts' },
+    { countryCode: 'th', name: 'Thailand - Chart of Accounts' },
```

Comment:
@mael-chouteau can you comment these out?

These Chart of Accounts aren't present ([ref](https://github.com/frappe/books/tree/master/fixtures/verified)), we can uncomment them as they're added.

Other than this everything else looks alright.